docs/source/installation/2-manual-install.rst

```diff
@@ -467,13 +464,13 @@ Install CommCare Cloud
 
        $ cp ~/commcare-cloud/src/commcare_cloud/config.example.py ~/commcare-cloud/src/commcare_cloud/config.py
 
-    Update the known hosts file
+    Update the known hosts file (substituting your environment name if necessary)
 
     ::
 
        $ commcare-cloud cluster update-local-known-hosts
```

Comment:
I know we have used cluster as environment name in the entire doc but would it be clearer if we use <env> which would indicate that this has to be replaced?
```suggestion
       $ commcare-cloud <env> update-local-known-hosts
```

btw I am totally fine with the way it is right now it was just a thought that I wanted to share with you.

Reply:
Yeah I do want to make this change. I also want to make a sweeping change to the `$` syntax before each command as it makes copy/pasting annoying, so was planning to do this in a separate PR if that is ok with you.

Reply:
Yeah. Totally fine by me.